tests/OSPSuite.Starter/Presenters/OptimizationStarterPresenter.cs

```diff
@@ -1,79 +1,41 @@
-﻿using System.Collections.Generic;
-using System.Data;
-using System.Drawing;
-using System.Windows.Forms;
-using OSPSuite.Core.Commands;
-using OSPSuite.Core.Domain;
+﻿using OSPSuite.Core.Commands;
 using OSPSuite.Core.Domain.Repositories;
 using OSPSuite.Core.Domain.Services.ParameterIdentifications;
-using OSPSuite.Presentation.Mappers.ParameterIdentifications;
-using OSPSuite.Presentation.Presenters;
 using OSPSuite.Presentation.Presenters.ParameterIdentifications;
-using OSPSuite.Presentation.Views.ParameterIdentifications;
 using OSPSuite.Starter.Tasks;
-using OSPSuite.Starter.Views;
-using OSPSuite.UI.Extensions;
+using OSPSuite.Starter.Tasks.Starters;
 using OSPSuite.Utility.Container;
 
 namespace OSPSuite.Starter.Presenters
 {
-   public interface IOptimizationStarterPresenter : IPresenter<IOptimizationStarterView>
+   public interface IOptimizationStarter : ITestStarter
    {
-      void StartMatrixTest();
-      void StartParameterIdentificationTest();
    }
 
-   public class OptimizationStarterPresenter : AbstractPresenter<IOptimizationStarterView, IOptimizationStarterPresenter>, IOptimizationStarterPresenter
+   public class OptimizationStarter : IOptimizationStarter
```

Comment:
I was fedup of always having to click twice for PI. Now the test is started automatically and Matrix test is  a separate button